src/test/java/duke/TaskTest.java

```diff
@@ -0,0 +1,26 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+public class TaskTest {
+    private static final String SAVEPATH = "./src/data/SaveData.txt";
```

Comment:
Might SAVE_PATH be a better name for this variable?

Reply:
Agree, the rest of your classes seem to use `SAVE_PATH` as well. Perhaps this is an oversight?